.github/workflows/release-tag-rc.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Get cleaned branch name
         id: clean_name
         run: |
-          REF_NAME=$(echo "${{ github.ref }}" | sed -e 's/refs\/heads\///' -e 's/refs\/tags\///' -e 's/release\/v//')
+          REF_NAME=$(echo "${{ github.ref }}" | sed -e 's/refs\/heads\///' -e 's/refs\/tags\/v//' -e 's/release\/v//')
```

Comment:
```suggestion
          REF_NAME=$(echo "${{ github.ref }}" | sed -e 's/refs\/heads\///' -e 's/refs\/tags\/v?//' -e 's/release\/v?//')
```

Reply:
Hmm. I think this is probably a fine change, but overall our tag/branch names should be consistent and this shouldn't be needed?

Reply:
That would be incorrect and require a re-tag either way imo

Reply:
Since the naming rule for tags and branches is acculate, I don't think it's necessary.